docs/serialization-guide.md

```diff
@@ -1,11 +1,11 @@
 # Kotlin Serialization Guide
 
-Kotlin Serialization is cross-platform and multi-format framework for data serialization &mdash;
-converting trees of objects to strings, byte arrays, or other _serial_ representations and back.
-Kotlin Serialization fully supports and enforces Kotlin type system, making sure only valid 
+Kotlin Serialization is a cross-platform and multi-format framework for data serialization&mdash;converting 
```

Comment:
Em dash, in contradiction to hyphen, should have spaces on moth sides

Reply:
Thanks for the feedback. I've always known it to be the opposite for formal writing:

* https://en.wikipedia.org/wiki/Dash#Spacing_and_substitution
* https://www.thepunctuationguide.com/em-dash.html#spaces
* https://www.quickanddirtytips.com/education/grammar/em-dash-space

The En dash is sometimes spaced, when used as an Em dash:

* https://en.wikipedia.org/wiki/Dash#Spacing

If this guide is meant to be formal, as you say in your other comment, I would think the unspaced Em dash would be correct. Still, I can change it back if you'd like.

Reply:
Ok then, thanks for clarifying that